app/src/app/components/spec-math-display-results/display-results.component.spec.ts

```diff
@@ -0,0 +1,100 @@
+// Copyright 2020 Google LLC
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+
+//     https://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { TestBed, ComponentFixture } from '@angular/core/testing';
+import { DisplayResultsComponent } from './display-results.component';
+
+import { MatTabsModule } from '@angular/material/tabs';
+import { BrowserModule } from '@angular/platform-browser';
+import { MatButtonModule } from '@angular/material/button';
+import { MatIconModule } from '@angular/material/icon';
+import { MatMenuModule } from '@angular/material/menu';
+import { MatDividerModule } from '@angular/material/divider';
+import { queryElement } from 'src/shared/functions';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+
+describe('DisplayResultsComponent', () => {
+  let fixture: ComponentFixture<DisplayResultsComponent>;
+  let component: DisplayResultsComponent;
+
+  beforeEach(async () => {
+    TestBed.configureTestingModule({
+      declarations: [
+        DisplayResultsComponent,
+      ],
+      imports: [
+        MatTabsModule,
+        BrowserModule,
+        MatButtonModule,
+        MatIconModule,
+        MatMenuModule,
+        MatDividerModule,
+        BrowserAnimationsModule
+      ],
+    }).compileComponents().then(() => {
+      fixture = TestBed.createComponent(DisplayResultsComponent);
+      component = fixture.componentInstance;
+    });
+  });
+
+  it('creates the DisplayResultsComponent', () => {
+    expect(component).toBeTruthy();
+  });
+
+  it('downloads the OperationSet zip file when the DOWNLOAD button is clicked', () => {
```

Comment:
maybe phrase more like 'invokes the downloadOperationSet' since it's not really testing the creation or download of the zip?

it would be nice if these tests could get at the functionality, but I can see that it might be difficult.  for example, maybe this test could have real dummy input files (if it doesn't already) and compare the zip output to a "golden" zip?  that would be potentially fragile though if the zip output were subject to change with, say, changes to the JSZip npm version.

Reply:
Will add that as a stretch goal for testing the downloads of the files 👍